tools/clang/include/clang/Sema/AttributeList.h

```diff
@@ -122,7 +122,7 @@ class AttributeList { // TODO: This should really be called ParsedAttribute
   /// True if this has a ParsedType
   unsigned HasParsedType : 1;
 
-  unsigned AttrKind : 8;
+  unsigned AttrKind : 16;
```

Comment:
To match with a change this will eventually collide with, It would be helpful if you used this fix instead:

```c++
  unsigned padding : 8;

  unsigned AttrKind;
```